phabricatoremails/render/templates/html/style.css

```diff
@@ -0,0 +1,403 @@
+.event-content {
```

Comment:
We should use Sass here or something similar to compile this CSS -- lots of hardcoded colours and values in here that can be defined more explicitly. Not a blocker, but this should go in a TODO.

Reply:
[Created a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1631956)